addons/l10n_vn/data/account_tax_report_data.xml

```diff
@@ -21,7 +21,7 @@
                     <record id="account_tax_report_line_01_01_vn" model="account.report.line">
                         <field name="name">VAT on purchase of goods and services</field>
                         <field name="code">VAT_ON_PURCHASE_OF_GOODS_AND_SERVICES</field>
-                        <field name="aggregation_formula">VAT_ON_PURCHASE_OF_GOODS_AND_SERVICES_0.balance + VAT_ON_PURCHASE_OF_GOODS_AND_SERVICES_5.balance + VAT_ON_PURCHASE_OF_GOODS_AND_SERVICES_10.balance</field>
+                        <field name="aggregation_formula">VAT_ON_PURCHASE_OF_GOODS_AND_SERVICES_0.balance + VAT_ON_PURCHASE_OF_GOODS_AND_SERVICES_5.balance + VAT_ON_PURCHASE_OF_GOODS_AND_SERVICES_10.balance + VAT_EXEMPTION_ON_PURCHASE_OF_GOODS_AND_SERVICES.balance</field>
                         <field name="children_ids">
                             <record id="account_tax_report_line_01_01_01_vn" model="account.report.line">
                                 <field name="name">VAT on purchase of goods and services 0%</field>
```

Comment:
Isn't it weird to have a `vat on exemption` in the `vat` section?
If this is an exemption, as that was pointed out, there isn't any vat possible. (not even 0%)

Reply:
@Levizar This is a tax report example in Vietnam call Form 01/GTGT VAT declaration which is enatced by Vietnam government (The picture is English version and the original one at: https://thuvienphapluat.vn/phap-luat-doanh-nghiep/bai-viet/mau-01-gtgt-to-khai-thue-gtgt-nam-2023-truong-hop-nop-thue-theo-phuong-phap-khau-tru-co-hoat-dong-san-xuat-kinh-doanh-2809.html)

![2024-04-16_18-04](https://github.com/odoo/odoo/assets/56789189/0b93d8dc-4f07-4016-b96f-54f73d5c1795)
As you can see from the picture that i have highlight some column which the `VAT EXEMPTION` is present which mean the both the `0% tax` and `VAT exempt` indeed have no tax amount but their appearance on the tax report is a must according to Vietnamese law . i hope this will help you understand and proceed in merging this PR

Reply:
@duong77476-viindoo Indeed, there isn't any VAT box for those exemption and the appearance of the printed tax report is irrelevant because (1) this pr isn't trying to reproduce the layout (see the screenshot below) and (2) it isn't the goal of Odoo's tax report to reproduce the exact layout a printed pdf would give.
I wanted to do such reproduction for the Danish VAT report at some point and I was given the same answer by our product owner.

I can ask them for their opinion if you want but I don't doubt it will be the same than they gave me 2 years ago.
 
![image](https://github.com/odoo/odoo/assets/52834046/c51cdb89-0cc7-4246-a379-d0f3f633bbb9)

Reply:
and looking closer at it, there shouldn't be any special line for the exempted purchase either. It seems its base should be included in the [23] tax grid. Only sales seems to have a specific tax grid for that.
![image](https://github.com/odoo/odoo/assets/52834046/c3e1ac69-31d1-4da1-93ea-861283a767a4)

Reply:
@Levizar Indeed you aren't wrong about this, in that case i have removed `VAT EXEMPTION` both for sale and purchase section only keep the `UNTAXED` one also remove related translation. I think we all good ?

Reply:
@duong77476-viindoo I think we are. I'm just doubting about the need for the untaxed base on purchase as I think it should be in tax grid 23 but not sure.

One of our BA is also checking but I think it will be good.
It will probably be merged this week.

Reply:
Good to hear thanks for the quick review, just ping me and i'll update in a flash :v:

Reply:
Hi @Levizar Does your BA finish checking ? i really hope this can be merged this week, thank you